js/components/operatorSignIn/operatorSignInModal.tsx

```diff
@@ -1,11 +1,73 @@
+import { post } from "../../api";
+import { findEmployeeByBadge, useEmployees } from "../../hooks/useEmployees";
 import { Modal } from "../modal";
 import { Attestation } from "./attestation";
+import { Error, Success } from "./complete";
 import { OperatorSelection } from "./operatorSelection";
-import { ReactElement, useState } from "react";
+import { DateTime } from "luxon";
+import { ReactElement, useEffect, useState } from "react";
+
+enum CompleteState {
+  SUCCESS,
+  ERROR,
+}
+
+export const submit = (
```

Comment:
**question:** does this actually need to be `export`-ed? I can't find any particular spots where it's used outside of this file.

Reply:
f1143a7